sources/packages/web/src/components/students/modals/PDStatusApplicationModal.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <Dialog
```

Comment:
mm, henry, as I shared we do have a generic component for modal called `ModalDialogBase`. you can refer the same example i shared 
![image](https://user-images.githubusercontent.com/77353155/156271646-9c1b7509-614f-4584-a6ab-b0f3b61a4b3d.png)

Reply:
this is primevue, we are using vuetify modal